src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerImpl.java

```diff
@@ -504,6 +565,32 @@ public void close() {
     }
     _kafkaConsumer.close();
```

Comment:
this would benefit greatly from a util ,ethod that would ensure all close() methods are called even in the presence of exceptions